prowler/lib/outputs/compliance/models.py

```diff
@@ -21,7 +21,34 @@ class Check_Output_MITRE_ATTACK(BaseModel):
     Requirements_SubTechniques: str
     Requirements_Platforms: str
     Requirements_TechniqueURL: str
-    Requirements_Attributes_AWSServices: str
+    Requirements_Attributes_Services: str
+    Requirements_Attributes_Categories: str
+    Requirements_Attributes_Values: str
+    Requirements_Attributes_Comments: str
+    Status: str
+    StatusExtended: str
+    ResourceId: str
+    CheckId: str
+    Muted: bool
+
+
+class Check_Output_MITRE_ATTACK_Azure(BaseModel):
```

Comment:
I'll rename this to comply with Python class naming convention.
```suggestion
class MitreAttackAzure(BaseModel):
```

Reply:
Done